packages/frontend/tests/acceptance/course/visualizations-instructor-test.js

```diff
@@ -88,8 +88,8 @@ module('Acceptance | course visualizations - instructor', function (hooks) {
     assert.strictEqual(page.root.sessionTypesChart.chart.slices.length, 2);
     assert.strictEqual(page.root.sessionTypesChart.chart.labels.length, 2);
     assert.strictEqual(page.root.sessionTypesChart.chart.descriptions.length, 2);
-    assert.strictEqual(page.root.sessionTypesChart.chart.labels[0].text, 'session type 1');
-    assert.strictEqual(page.root.sessionTypesChart.chart.labels[1].text, 'session type 0');
+    assert.ok(page.root.sessionTypesChart.chart.labels[0].text.includes('session type 1'));
+    assert.ok(page.root.sessionTypesChart.chart.labels[1].text.includes('session type 0'));
```

Comment:
Reading the commit message I wonder if [String.startsWith](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith) might be a better choice for these?

Reply:
sounds good, will make that change.

Reply:
done :heavy_check_mark: